starter_code/views/index.hbs

```diff
@@ -1,2 +1,8 @@
-<h1>{{title}}</h1>
+<div class="index">
+
+<div class="text"><h1>{{title}}</h1>
 <p>Welcome to {{title}}</p>
+<button><a href="/movies">Check the movies !</a></button>
```

Comment:
it's not good practice to put an a tag (for linking) inside a button (for doing things, like submitting) because they are used for different things.  You'll notice that when you click the button and not on the text it only highlights the button. So you should use an a tag and style it to look like a a button